src/theory/quantifiers/sygus/sygus_interpol.cpp

```diff
@@ -319,6 +319,13 @@ bool SygusInterpol::solveInterpolation(const std::string& name,
                                        const TypeNode& itpGType,
                                        Node& interpol)
 {
+  collectSymbols(axioms, conj);
```

Comment:
Can you give a quick comment on what this is done here (based on what you said in the PR)?

Reply:
OK. I wrote an explanation for reordering the instructions. Do you think it's clear?